app/configuration/Config.scala

```diff
@@ -27,8 +27,6 @@ object Config {
 
   val playSecret = config.getString("play.crypto.secret")
 
-  lazy val googleAuthConfig = googleAuthConfigFor(config)
```

Comment:
this has moved to be dependency injected.  to be honest if we're using DI I'd prefer to get rid of config object and just have separate case classes which each area of config only injected where relevant